openstack-manager-blueprint.yaml

```diff
@@ -413,7 +423,7 @@ inputs:
   # below modules require compilation.
   install_python_compilers:
     type: string
-    default: 'true'
+    default: ''
```

Comment:
did we decide to change this to empty? doesn't that create a problem on bootstrap? let's just verify that